### PR TITLE
docs(website): update pattern contribution guidelines

### PR DESCRIPTION
**Pull request by corinne-nagel**
Updating the contribution to patterns guidance

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by TheSisb — APPROVED** [review]

**Review by richbachman — APPROVED** [review]

**Comment by katieporter** [comment]
Looks great! Just noticed a couple things:
- Could we get rid of the colons at the end of headings? They just look a little funky with the # icon next to them. For the last heading "Contributions can look like", maybe that heading is appended with an ellipses to match the "Paste is...", "Paste is not..." headings on the [about](https://paste.twilio.design/getting-started/about-paste) page. (I just updated this in your [Google doc](https://docs.google.com/document/d/1NSJioErOizis42Lfri4dsxiM_m9ObWUDGfrmZj9yH-g/edit?usp=sharing) so you can see what I mean.)
- In the very last bullet point, "Patterns are for you..." should be on its own line, not inside the list.
- The anchor inside of the callout at the bottom isn't working. I think you'll need to use the Anchor component rather than the Markdown that you used, because it's inside of the Callout component. You can copy markup from the callout on the Patterns overview page.

File: packages/paste-website/src/pages/patterns/contribute/index.mdx [hunk 1]
```diff
@@ -63,84 +63,48 @@ export const pageQuery = graphql`
...
-
-## Updating an existing pattern
-
-When you want to make an update to an existing pattern (be it a beta pattern, or a production pattern),
-you will follow many of the same steps as you would when contributing a new pattern, depending on the
-degree of the change. Reach out to us in `#help-design-system` if you’d like to update an existing pattern,
-and we can get the conversation started.
+Thanks for your interest in contributing patterns! Since this is a general guideline, feel free to grow or shrink this process however you need.
+
+## General process:
```
Comment (katieporter) [inline packages/paste-website/src/pages/patterns/contribute/index.mdx:68]:
```suggestion
## General process
```

File: packages/paste-website/src/pages/patterns/contribute/index.mdx [hunk 2]
```diff
@@ -63,84 +63,48 @@ export const pageQuery = graphql`
...
+  
+  - Documenting one part of a pattern
+  
+  - Sharing research related to a pattern
+  
+  - Refining an existing pattern based on new information
+
+### Where do I start?
+
+Check out our [list of most requested patterns](https://paste.twilio.design/patterns#patterns) to see what's available. <strong>Have a new idea?</strong> Reach out to your peers and the `#help-design-system` channel to see if there’s a need for something else.
```
Comment (katieporter) [inline packages/paste-website/src/pages/patterns/contribute/index.mdx:107]:
```suggestion
Check out our [list of most requested patterns](/patterns#patterns) to see what's available. <strong>Have a new idea?</strong> Reach out to your peers and the `#help-design-system` channel to see if there’s a need for something else.
```

Comment (katieporter) [inline packages/paste-website/src/pages/patterns/contribute/index.mdx:107]:
```suggestion
Check out our [list of most requested patterns](https://paste.twilio.design/patterns#patterns) to see what's available. <strong>Have a new idea?</strong> Reach out to your peers and create a [Github Discussion](https://github.com/twilio-labs/paste/discussions) to see if there’s a need for something else.
```

File: packages/paste-website/src/pages/patterns/contribute/index.mdx [hunk 3]
```diff
@@ -63,84 +63,48 @@ export const pageQuery = graphql`
...
+  
+  - Documenting one part of a pattern
+  
+  - Sharing research related to a pattern
+  
+  - Refining an existing pattern based on new information
+
+### Where do I start?
+
+Check out our [list of most requested patterns](https://paste.twilio.design/patterns#patterns) to see what's available. <strong>Have a new idea?</strong> Reach out to your peers and create a [Github Discussion](https://github.com/twilio-labs/paste/discussions) to see if there’s a need for something else.
```
Comment (katieporter) [inline packages/paste-website/src/pages/patterns/contribute/index.mdx:107]:
```suggestion
Check out our [list of most requested patterns](/patterns#patterns) to see what's available. <strong>Have a new idea?</strong> Reach out to your peers and create a [Github Discussion](https://github.com/twilio-labs/paste/discussions) to see if there’s a need for something else.
```

**Review by katieporter — APPROVED** [review]

**Pull request merged**